gnd/src/main/res/layout/fragment_terms_service.xml

```diff
@@ -47,29 +65,27 @@
       android:fillViewport="true"
       android:scrollbars="vertical"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/toolbar">
-
+      app:layout_constraintTop_toBottomOf="@id/terms_header">
         <TextView
           android:id="@+id/termsText"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_margin="8dp"
           android:text="@string/tos_title"
-          android:visibility="visible" />
+          android:visibility="visible"
```

Comment:
The title currently appears both in the toolbar and scrollview. Please remove duplicate title here.

Reply:
Due to same string resource assigned but termsText will be updated from remote db. updated in #845